sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1442,13 +1448,17 @@ static void _event_cb(netdev_t *dev, netdev_event_t event)
         switch (event) {
             case NETDEV_EVENT_RX_COMPLETE:
                 pkt = netif->ops->recv(netif);
+                /* send packets previously queued within netif due to the lower
```

Comment:
wording nitpick here and below: only one packet is sent (not packet*s*).

Reply:
Fixed